tests/solvers/test_base_solver.py

```diff
@@ -0,0 +1,65 @@
+from typing import Tuple
+
+from scipy.sparse import csr_matrix
+import pytest
+
+import numpy as np
+
+from tests._utils import ATOL, RTOL, MockSolverOutput
+
+
+class TestBaseSolverOutput:
+    @pytest.mark.parametrize("batch_size", [1, 4])
+    @pytest.mark.parametrize("threshold", [0.0, 1e-1, 1.0])
+    @pytest.mark.parametrize("shape", [(7, 2), (91, 103)])
+    def test_compute_sparsification_threshold(self, batch_size: int, threshold: float, shape: Tuple[int, int]) -> None:
+        rng = np.random.RandomState(42)
+        tmap = np.abs(rng.rand(shape[0], shape[1]))
+        tmap = tmap / tmap.sum()
```

Comment:
Why are you normalizing here? If not necessary, would remove.

Reply:
To make it a bit more similar to a transport matrix, but can also remove